[2024-12-12T09:09:52.289+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-12-12T09:09:52.312+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dbt_orchestrator.test_connection.dbt_test_connection scheduled__2024-12-12T08:45:00+00:00 [queued]>
[2024-12-12T09:09:52.323+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dbt_orchestrator.test_connection.dbt_test_connection scheduled__2024-12-12T08:45:00+00:00 [queued]>
[2024-12-12T09:09:52.323+0000] {taskinstance.py:2306} INFO - Starting attempt 3 of 3
[2024-12-12T09:09:52.336+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): test_connection.dbt_test_connection> on 2024-12-12 08:45:00+00:00
[2024-12-12T09:09:52.342+0000] {standard_task_runner.py:63} INFO - Started process 2838 to run task
[2024-12-12T09:09:52.345+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dbt_orchestrator', 'test_connection.dbt_test_connection', 'scheduled__2024-12-12T08:45:00+00:00', '--job-id', '408', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpthtpuikv']
[2024-12-12T09:09:52.346+0000] {standard_task_runner.py:91} INFO - Job 408: Subtask test_connection.dbt_test_connection
[2024-12-12T09:09:52.395+0000] {task_command.py:426} INFO - Running <TaskInstance: dbt_orchestrator.test_connection.dbt_test_connection scheduled__2024-12-12T08:45:00+00:00 [running]> on host 34fdc836a0bb
[2024-12-12T09:09:52.488+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ma7moud' AIRFLOW_CTX_DAG_ID='dbt_orchestrator' AIRFLOW_CTX_TASK_ID='test_connection.dbt_test_connection' AIRFLOW_CTX_EXECUTION_DATE='2024-12-12T08:45:00+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-12T08:45:00+00:00'
[2024-12-12T09:09:52.489+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-12-12T09:09:52.501+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-12-12T09:09:52.501+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'dbt debug --profiles-dir /opt/***/dbt --project-dir /opt/***/dbt/ecommerce']
[2024-12-12T09:09:52.514+0000] {subprocess.py:86} INFO - Output:
[2024-12-12T09:09:54.593+0000] {subprocess.py:93} INFO - [0m09:09:54  Running with dbt=1.8.0
[2024-12-12T09:09:54.606+0000] {subprocess.py:93} INFO - [0m09:09:54  dbt version: 1.8.0
[2024-12-12T09:09:54.606+0000] {subprocess.py:93} INFO - [0m09:09:54  python version: 3.9.19
[2024-12-12T09:09:54.607+0000] {subprocess.py:93} INFO - [0m09:09:54  python path: /home/***/.local/bin/python
[2024-12-12T09:09:54.607+0000] {subprocess.py:93} INFO - [0m09:09:54  os info: Linux-6.8.0-49-generic-x86_64-with-glibc2.36
[2024-12-12T09:09:54.683+0000] {subprocess.py:93} INFO - [0m09:09:54  Using profiles dir at /opt/***/dbt
[2024-12-12T09:09:54.684+0000] {subprocess.py:93} INFO - [0m09:09:54  Using profiles.yml file at /opt/***/dbt/profiles.yml
[2024-12-12T09:09:54.684+0000] {subprocess.py:93} INFO - [0m09:09:54  Using dbt_project.yml file at /opt/***/dbt/ecommerce/dbt_project.yml
[2024-12-12T09:09:54.685+0000] {subprocess.py:93} INFO - [0m09:09:54  adapter type: postgres
[2024-12-12T09:09:54.686+0000] {subprocess.py:93} INFO - [0m09:09:54  adapter version: 1.8.0
[2024-12-12T09:09:54.773+0000] {subprocess.py:93} INFO - [0m09:09:54  [[33mWARNING[0m]: Deprecated functionality
[2024-12-12T09:09:54.774+0000] {subprocess.py:93} INFO - The `tests` config has been renamed to `data_tests`. Please see
[2024-12-12T09:09:54.774+0000] {subprocess.py:93} INFO - https://docs.getdbt.com/docs/build/data-tests#new-data_tests-syntax for more
[2024-12-12T09:09:54.774+0000] {subprocess.py:93} INFO - information.
[2024-12-12T09:09:54.827+0000] {subprocess.py:93} INFO - [0m09:09:54  Configuration:
[2024-12-12T09:09:54.828+0000] {subprocess.py:93} INFO - [0m09:09:54    profiles.yml file [[32mOK found and valid[0m]
[2024-12-12T09:09:54.829+0000] {subprocess.py:93} INFO - [0m09:09:54    dbt_project.yml file [[32mOK found and valid[0m]
[2024-12-12T09:09:54.829+0000] {subprocess.py:93} INFO - [0m09:09:54  Required dependencies:
[2024-12-12T09:09:54.840+0000] {subprocess.py:93} INFO - [0m09:09:54   - git [[32mOK found[0m]
[2024-12-12T09:09:54.840+0000] {subprocess.py:93} INFO - 
[2024-12-12T09:09:54.840+0000] {subprocess.py:93} INFO - [0m09:09:54  Connection:
[2024-12-12T09:09:54.841+0000] {subprocess.py:93} INFO - [0m09:09:54    host: 172.18.0.3
[2024-12-12T09:09:54.841+0000] {subprocess.py:93} INFO - [0m09:09:54    port: 5432
[2024-12-12T09:09:54.842+0000] {subprocess.py:93} INFO - [0m09:09:54    user: ***
[2024-12-12T09:09:54.842+0000] {subprocess.py:93} INFO - [0m09:09:54    database: ecommerce
[2024-12-12T09:09:54.843+0000] {subprocess.py:93} INFO - [0m09:09:54    schema: olap_model
[2024-12-12T09:09:54.843+0000] {subprocess.py:93} INFO - [0m09:09:54    connect_timeout: 10
[2024-12-12T09:09:54.844+0000] {subprocess.py:93} INFO - [0m09:09:54    role: None
[2024-12-12T09:09:54.845+0000] {subprocess.py:93} INFO - [0m09:09:54    search_path: None
[2024-12-12T09:09:54.846+0000] {subprocess.py:93} INFO - [0m09:09:54    keepalives_idle: 0
[2024-12-12T09:09:54.846+0000] {subprocess.py:93} INFO - [0m09:09:54    sslmode: None
[2024-12-12T09:09:54.847+0000] {subprocess.py:93} INFO - [0m09:09:54    sslcert: None
[2024-12-12T09:09:54.847+0000] {subprocess.py:93} INFO - [0m09:09:54    sslkey: None
[2024-12-12T09:09:54.848+0000] {subprocess.py:93} INFO - [0m09:09:54    sslrootcert: None
[2024-12-12T09:09:54.848+0000] {subprocess.py:93} INFO - [0m09:09:54    application_name: dbt
[2024-12-12T09:09:54.849+0000] {subprocess.py:93} INFO - [0m09:09:54    retries: 1
[2024-12-12T09:09:54.850+0000] {subprocess.py:93} INFO - [0m09:09:54  Registered adapter: postgres=1.8.0
[2024-12-12T09:09:55.013+0000] {subprocess.py:93} INFO - [0m09:09:55    Connection test: [[32mOK connection ok[0m]
[2024-12-12T09:09:55.013+0000] {subprocess.py:93} INFO - 
[2024-12-12T09:09:55.014+0000] {subprocess.py:93} INFO - [0m09:09:55  [32mAll checks passed![0m
[2024-12-12T09:09:56.000+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-12-12T09:09:56.001+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-12-12T09:09:56.024+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dbt_orchestrator, task_id=test_connection.dbt_test_connection, run_id=scheduled__2024-12-12T08:45:00+00:00, execution_date=20241212T084500, start_date=20241212T090952, end_date=20241212T090956
[2024-12-12T09:09:56.048+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-12-12T09:09:56.083+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-12T09:09:56.084+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
